javatests/arcs/core/entity/BUILD

```diff
@@ -41,8 +42,14 @@ arcs_kt_jvm_library(
         ["*.kt"],
         exclude = TEST_SRCS,
     ),
+    visibility = ["//visibility:public"],
```

Comment:
You shouldn't need this / this is a bad idea. This library should only be used inside this package. If it's supposed to be a more general purpose library, it should live in //java/.../testing or //java/.../testutil, and be marked `testonly = True`

Reply:
Thanks. I wasn't sure how to handle this, or why it wasn't visible. This file isn't test utilities, it's actual tests that get run in different configurations, so putting it in `//java` doesn't feel quite right.